src/templates/base/2019/base.html

```diff
@@ -274,6 +268,18 @@
             }
           });
         }
+        // Year switching
+        var yearSwitchers = document.querySelectorAll('.year-switcher select');
+        for (var i = 0; i < yearSwitchers.length; i++) {
```

Comment:
The bodies of these for loops (lang and year) are functionally the same. Could we:

- write a `querySelectorAll` to include both lang and year switchers
- rewrite the loop variables to refer to generic switchers as opposed to year/lang-specific

Reply:
Fair point. Done.